src/mscorlib/src/System/Reflection/Emit/ILGenerator.cs

```diff
@@ -1174,6 +1174,25 @@ public virtual void ThrowException(Type excType)
             this.Emit(OpCodes.Throw);
         }
 
+        private static Type GetConsoleType()
+        {
+#if PLATFORM_UNIX
+            Type t = null;
+            try
+            {
+                t = Type.GetType("System.Console, System.Console, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a", throwOnError: false);
+            }
+            catch { } // even with throwOnError:false, could still throw for a variety of reasons
```

Comment:
I agree. I think some mythology about how to write calls to Type.GetType spread while running in to issues on DNX prior to this fix: https://github.com/jkotas/coreclr/commit/8d6ac01710c84c93c389eb1212c29052fb33475e